src/components/ReserveForm/ReserveForm.jsx

```diff
@@ -129,15 +135,14 @@ const ReserveForm = ({ venues, id_restaurant }) => {
             className="reserve-form-input"
           >
             <option value="CHOOSE ONE">CHOOSE A VENUE</option>
-            {
-              !!venues.map((venue, index) => {
+            {!!venues &&
+              venues.map((venue, index) => {
```

Comment:
this change was proposed in a previous PR and it doesn't work because the map doesn't get executed, therefore it needs to pass the check to then execute the map